pom.xml

```
@@ -104,7 +104,7 @@
         <antlr.version>4.7.2</antlr.version>
 
         <!-- Quarkus -->
-        <quarkus.version>1.12.0.Final</quarkus.version>
+        <quarkus.version>2.0.0.CR2</quarkus.version>
```

```suggestion
        <quarkus.version>2.0.0.CR3</quarkus.version>
```

2.0.0.CR3 is available since a couple of hours.

---

Good point, I'll adjust the PR to use CR3 since that's now available.